src/components/side_panel/side_panel/side_panel.xml

```diff
@@ -1,20 +1,21 @@
 <templates>
   <t t-name="o-spreadsheet-SidePanel" owl="1">
-    <div class="o-spreadsheet-side-panel-container">
+    <div
+      class="o-spreadsheet-side-panel-container d-flex flex-column bg-white border user-select-none">
       <div
-        class="o-spreadsheet-side-panel-header d-flex align-items-center justify-content-between border-top border-bottom fw-bold p-3">
+        class="o-spreadsheet-side-panel-header d-flex align-items-center justify-content-between border-bottom fw-bold p-3">
```

Comment:
The style is not so pretty after this change and it' s not that easy to solve because the topbar border seems to be overlapping on the borders of the side panel.

I'd argue that we should keep border-top border-bottom atm as well a forcing a darker border (like `border-secondary`) both on this div and its parent  `o-spreadsheet-side-panel-container` to keep in touch with the current style

Also you dropped the grey background here in the previous commit (b9e01bdce8a46141b7f41f5d283d5f1c9c6b3fff) d on't know if it's on purpose ?

![Screenshot from 2022-11-08 12-47-15](https://user-images.githubusercontent.com/10333790/200556306-1c17018e-5f2c-4be1-b973-e013a9c0eecd.png)